en/docs/guides/organization-login/try-organization-login.md

```diff
@@ -182,6 +190,9 @@ To set up the sample application:
     ]
     ```
 
+    !!! info
+        If the created application resides in a different organization, change the value in the ```WSO2IS_TENANT_NAME``` to that organization id.
```

Comment:
If the users are just reading through the doc before testing this out, they might wonder where `WSO2IS_TENANT_NAME` is. 

We better add it to the sample.

Reply:
Sure, I will update the document

Reply:
Updated the PR.